store/tikv/client.go

```diff
@@ -152,6 +153,7 @@ func (c *batchCommandsClient) batchRecvLoop(cfg config.TiKVClient) {
 				// TODO: Use a more smart backoff strategy.
 				time.Sleep(time.Second)
 			}
+			metrics.TiKVBatchClientUnavailable.Observe(time.Since(now).Seconds())
```

Comment:
If the previous loop cannot stop, you will never get here.

Reply:
If it never break the loop, tidb is unavailable... we'll know that by other metrics such as QPS may drop to zero @shenli